nailgun/entities.py

```diff
@@ -2909,6 +2909,43 @@ def __init__(self, server_config=None, **kwargs):
         }
         super().__init__(server_config=server_config, **kwargs)
 
+    def remove_version(self, versions, synchronous=True, timeout=None):
+        """Remove content view Version(s) from this content view. 
+            Will first remove the CV from the Version's environment(s).
+
+            :param versions: ContentViewVersion (entity) or ID (int) to remove.
+                can also pass a list of entities, or a list of IDs.
+        """
+        # Normalize into a list of ids; from a single entity/:id, or a list.
+        if isinstance(versions, list):
+            version_ids = [v.id if hasattr(v, 'id') else int(v) for v in versions]
+        else:
+            version_ids = [versions.id if hasattr(versions, 'id') else int(versions)]
+
+        cv = self.read()
+        matched_versions = []
+        # Match the Version ID from CV to the provided Versions
+        for vid in version_ids:
+            matched_versions = [v for v in cv.version if v.id == vid]
+            if not matched_versions:
+                continue
+            for version in matched_versions:
+                # delete CV from env, for any env contained in matched Versions
+                for env in version.read().environment:
+                    self.delete_from_environment(env.id)
+        if not matched_versions:
+            raise ValueError(
+                f'No Version(s) provided: {versions} ,'
+                f' matched the published Versions of the Content-View[id:{cv.id}]: {cv.version}')
+
+        response = client.put(
+            f'{self.path('remove')}',
```

Comment:
This is what the checks are complaining about. Should work with newer python versions, but not 3.11
```suggestion
            f'{self.path("remove")}',
```

Reply:
I had to reset my venv few days ago and was missing the file for Nailgun internal pre-commit.
admittedly committed and pushed this draft with `--no-verify` lol